test/tap/tests/Makefile

```diff
@@ -12,6 +12,7 @@ PROXYSQL_PATH=../../..
 PROXYSQL_IDIR=$(PROXYSQL_PATH)/include
```

Comment:
In the system I'm using for building, an Ubuntu 19.10, an additional dependency is required for building the tests: **libconfig++-dev**, maybe this is something we should reflect in the doc. Also, for executing the scripts for the tests, installing the module _TAP::Parser::SourceHandler::MyTAP_ is also required, maybe this should be reflected too.

Reply:
libconfig++-dev Fixed

Reply:
Fixed